cvat-ui/src/components/task-page/styles.scss

```diff
@@ -8,7 +8,6 @@
     overflow-y: auto;
     height: 90%;
     width: 100%;
-    position: fixed;
```

Comment:
As far as I remember layout in Safary without this css property is broken.

Reply:
It brings an unexpected behavior in the screen size < 1280 in other cases with x scrolling of header

Reply:
Any way it should be solved another way in my opinion

Reply:
I agree that there is probably better way, but nevertheless, need to find it first and implement instead. 
I think we should not push to develop changes that makes UI experience worse.

Reply:
I just check this in Safari. What exactly doesn't works in it?

![image](https://user-images.githubusercontent.com/2045615/94604057-cc905a00-029f-11eb-81a6-aeaad9025f38.png)

![image](https://user-images.githubusercontent.com/2045615/94604127-e762ce80-029f-11eb-920f-34b4960b147b.png)

![image](https://user-images.githubusercontent.com/2045615/94604230-082b2400-02a0-11eb-803e-3a2390ea6b4a.png)

![image](https://user-images.githubusercontent.com/2045615/94604272-19743080-02a0-11eb-841c-a8d61a4476bf.png)

![image](https://user-images.githubusercontent.com/2045615/94604321-2bee6a00-02a0-11eb-9065-ef9c8cefcc89.png)

Reply:
I cannot check it right now (do not have a mac near with me). But if it works, okay. As far as I remember there were something wrong with layout (maybe with a background color and scrolling, especially on tasks view)